governance.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Governance 1.0 DRAFT WIP
+description: Governance 1.0 discussion draft
+---
+# Governance 1.0 `DRAFT WIP`
+
+This document contains a work in progress (WIP) draft of the initial governance of the Op1st project (Governance 1.0).
+
+It also has a roadmap for additions coming in the next iteration of governance (Governane 1.1).
+
+## Op1st Community Governance 1.0 (MVG)
+
+A healthy open source project is one that demonstrates open practices, uses open infrastructure, and cultivates an open culture with the goal of becoming more sustainable.
+
+### Principles
+As a new cloud-native community growing up at the footsteps of massively scaled and successful projects such as Kubernetes and OpenInfra, we want to be careful to learn from those who came before us, while forging our own path into the challenges we've chosen to undertake.
+
+To stand up this Governance for a 1.0 good-enough minimal-viable-governance (MVG), we are looking to our key predecessors for some of our starting points.
+As an initial set of principles we are combining elements of the [Kubernetes community principles](https://github.com/kubernetes/community/blob/master/governance.md#principles) and the [OpenInfra Foundations Four Opens](https://openinfra.dev/four-opens/).
+
+#### Op1st Community Principles 1.0
+Our mission requires us to use open source principles in ways beyond licensing the code we write, but at the core we are an open source community and want to adhere to the best possible principles for creating an open, welcoming, and inclusive community.
+
+These are the Op1st community principles we adhere to:
+1. **Open Source** : Any content we create is released under an open source license. Content includes software, documentations, configuration files, and training materials.
```

Comment:
Nit: Do we need to define what "content" is here? If so, we should be more inclusive I think:

```suggestion
1. **Open Source** : Any content we create is released under an open source license. Content includes software, documentations, configuration files, deployment manifests, telemetry data, training materials, and any other content meaningful to the project.
```

Reply:
Love your use of the Suggestion feature! I just learned about that today.

I did a suggestion [above](https://github.com/operate-first/community/pull/107/files/092e770de008cf2c8ccfd95f528b3326618d2baa#r771018629), and I think it works to cover your use cases -- configuration should cover deployment manifests (and if not then we can keep it), and data covers all data.

I also think I'm going to change my fix to "includes but is not limited to" as a different way to include things not listed.

Reply:
I like that :+1: :slightly_smiling_face: